src/BigDecimal.php

```diff
@@ -23,7 +23,7 @@ final class BigDecimal extends BigNumber
      * No leading zero must be present.
      * No leading minus sign must be present if the value is 0.
      *
-     * @var string
+     * @var numeric-string
```

Comment:
My IDE (PHPStorm) does not know the non-standard `numeric-string` type. Is there a way to make both PHPStorm & Psalm happy, by keeping the `@var string` annotation, but adding Psalm-specific annotations?

Reply:
I will do this via `@psalm-var` annotation